lang/en.json

```diff
@@ -2989,6 +2989,12 @@
   "THEME": {
     "Name": "Theme",
     "Hint": "Theme that will apply to the UI and all sheets by default. Automatic will be determined by your browser or operating system settings."
+  },
+  "RULESVERSION": {
+    "Name": "Rules Version",
+    "Hint": "Change handling of various rules (including exhausion, Jack of All Trades, Remarkable Athlete, and hit dice recovery) between the 2024 and 2014 rule sets.",
```

Comment:
I don't know if we really need to name rules here. I'm sure the list will grow (we're missing Wild Shape, for example), and then these particular choices will seem arbitrary.